gohai.go

```diff
@@ -20,16 +22,24 @@ type Collector interface {
 	Collect() (interface{}, error)
 }
 
+type SelectedCollectors map[string]bool
+
 var collectors = []Collector{
 	&cpu.Cpu{},
 	&filesystem.FileSystem{},
 	&memory.Memory{},
 	&network.Network{},
 	&platform.Platform{},
+	&processes.Processes{},
 }
 
-var options struct {
+var options = struct {
+	only    SelectedCollectors
+	exclude SelectedCollectors
 	version bool
+}{
+	only:    make(SelectedCollectors),
+	exclude: make(SelectedCollectors),
 }
```

Comment:
for clarity, maybe just define the options struct here, and initialize in `init()`